### PR TITLE
1559-add-citieskg-namespace-as-derivation-input: Added code to add ci…

### PR DESCRIPTION
**Pull request by nagarajankarthik**
Added citieskg namespace as input to the dispersion derivation.

### CODE REVIEW

File: JPS_VIRTUALSENSOR/AermodAgent/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   aermod-agent:
-    image: ghcr.io/cambridge-cares/aermod-agent:1.1.2-SNAPSHOT
+    image: ghcr.io/cambridge-cares/aermod-agent:1.1.2-1559-add-citieskg-namespace-as-derivation-input-SNAPSHOT
```
Comment (kok-foong) [inline JPS_VIRTUALSENSOR/AermodAgent/docker-compose.yml:5]: remember to increment the number

File: JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java [hunk 2]
```diff
@@ -56,6 +56,26 @@ public void processRequestParameters(DerivationInputs derivationInputs, Derivati
...
+
+        String citiesNamespace = null;
+
+
+        if (citiesNamespaceIri != null) {
+            citiesNamespace = queryClient.getCitiesNamespace(citiesNamespaceIri);
+        } else {
+            LOGGER.info("No citieskg namespace was specified in the POST request to Dispersion Interactor." + 
+            "Static point sources will not be included in this AERMOD run.");
+        }
```
Comment (kok-foong) [inline JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java:78]:
```suggestion
        String citiesNamespace= null;
        if (derivationInputs.getIris(QueryClient.CITIES_NAMESPACE) != null) {
            String citiesNamespaceIri = derivationInputs.getIris(QueryClient.CITIES_NAMESPACE).get(0);
            citiesNamespace = queryClient.getCitiesNamespace(citiesNamespaceIri);
        } else {
            LOGGER.info("No citieskg namespace was specified in the POST request to Dispersion Interactor." + 
            "Static point sources will not be included in this AERMOD run.");
        }
```

File: JPS_VIRTUALSENSOR/AermodAgent/docker-compose.yml (hunk 1 above)
Comment (nagarajankarthik) [reply JPS_VIRTUALSENSOR/AermodAgent/docker-compose.yml]: The number has been changed to 1.1.3 in the latest commit in this docker-compose.yml and also the startup.sh and aermod-agent.json file in the stack-manager directory.

File: JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java (hunk 2 above)
Comment (nagarajankarthik) [reply JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java]: Done. Please see latest commit.

Comment (nagarajankarthik) [reply JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java]: I only tested my version that checks for the exception but not this version that checks for the null value.

File: JPS_VIRTUALSENSOR/DispersionInteractor/src/main/java/com/cmclinnovations/dispersion/InitialiseSimulation.java [hunk 3]
```diff
@@ -57,6 +57,10 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         String ewkt = req.getParameter("ewkt");
         int nx = Integer.parseInt(req.getParameter("nx"));
         int ny = Integer.parseInt(req.getParameter("ny"));
+        String citiesNamespace = null;
+        if (req.getParameter("citiesnamespace") != null){
+            citiesNamespace = req.getParameter("citiesnamespace");
+        }
```
Comment (kok-foong) [inline JPS_VIRTUALSENSOR/DispersionInteractor/src/main/java/com/cmclinnovations/dispersion/InitialiseSimulation.java:63]:
```suggestion
        String citiesNamespace = req.getParameter("citiesnamespace");
```

Comment (kok-foong) [reply JPS_VIRTUALSENSOR/DispersionInteractor/src/main/java/com/cmclinnovations/dispersion/InitialiseSimulation.java:63]: I believe this should give the same result

Comment (nagarajankarthik) [reply JPS_VIRTUALSENSOR/DispersionInteractor/src/main/java/com/cmclinnovations/dispersion/InitialiseSimulation.java]: Done.

File: JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java [hunk 4]
```diff
@@ -89,6 +89,7 @@ public class QueryClient {
     public static final String NX = PREFIX_DISP + "nx";
     public static final String NY = PREFIX_DISP + "ny";
     public static final String SCOPE = PREFIX_DISP + "Scope";
+    public static final String CITIES_NAMESPACE = "https://www.theworldavatar.com/kg/ontodispersion/CitiesNamespace";
```
Comment (kok-foong) [inline JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java:92]: Changed to https://www.theworldavatar.com/kg/ontodispersion/OntoCityGMLNamespace, should merge the TBox soon

Comment (nagarajankarthik) [reply JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java]: Done.

**Pull request merged**